addons/crates_c/Crates/Crate_Ammo.hpp

```diff
@@ -469,5 +469,6 @@
         	MACRO_ADDMAGAZINE(16aa_cone,30);
         	MACRO_ADDMAGAZINE(16aa_sign_road_closed,15);
         	MACRO_ADDMAGAZINE(16aa_sign_checkpoint,15);
+            MACRO_ADDITEM(ACE_Sandbag_empty,50);
```

Comment:
indentation is not consistent with the others. This looks like an exessive space